### PR TITLE
PBS-679 Add comparable function for flightAuthDutyCode

### PR DESCRIPTION
**Pull request by Pete-Hoch**
Add struct dutyCodeCompInfo and refactored comparable code from both CrewLine and AircrewData into one func.

### CODE REVIEW

File: PBLogging/Data/EntityExtensions/CrewLine+Comparable.swift [hunk 1]
```diff
@@ -8,9 +8,39 @@
 extension CrewLine: Comparable {
 
     public static func < (lhs: CrewLine, rhs: CrewLine) -> Bool {
+        return dutyCodeLessThan(dutyCodeCompInfo(lhs), dutyCodeCompInfo(rhs))
+    }
+
+    public struct dutyCodeCompInfo {
```
Comment (jbethanco) [inline PBLogging/Data/EntityExtensions/CrewLine+Comparable.swift:14]: Struct name should be capitalized.

**Review by jbethanco — APPROVED** [review]
Not sure why but the sorting isn't matching up on the forms and on the 

In AircrewListView changed line 65 to:
ForEach(sortie.crewLines.sorted()) { crewLine in

The sort still isn't matching the generated 781.

Not sure what is/isn't happening.

File: PBLogging/Data/EntityExtensions/CrewLine+Comparable.swift (hunk 1 above)
Comment (Pete-Hoch) [reply PBLogging/Data/EntityExtensions/CrewLine+Comparable.swift]: Opps, fixed.

**Comment by Pete-Hoch** [comment]
> Not sure why but the sorting isn't matching up on the forms and on the AircrewListView.
> The sort still isn't matching the generated 781.
> Not sure what is/isn't happening.

Looking at SpaceX Recovery and the first Sortie, I see the sorted Aircrew as the same. So is it just some?

**Comment by jbethanco** [comment]
> > Not sure why but the sorting isn't matching up on the forms and on the AircrewListView.
> > The sort still isn't matching the generated 781.
> > Not sure what is/isn't happening.
> 
> Looking at SpaceX Recovery and the first Sortie, I see the sorted Aircrew as the same. So is it just some?

Not sure, did on FEMA mission. We should probably write a test on CrewLine vs AircrewData, using the same data and then sort each and compare arrays just to be certain. I'm wondering if in my rush to get out the 781 generation I forgot to copy over names or something into the aircrewdata from the crewlines.

**Pull request merged**